pepys_import/core/store/data_store.py

```diff
@@ -44,6 +45,23 @@
 DEFAULT_DATA_PATH = os.path.join(PEPYS_IMPORT_DIRECTORY, "database", "default_data")
 USER = getuser()  # Login name of the current user
 
+# Constant Lists of revisions - list all files in the directory
+SQLITE_REVISIONS_FOLDER = os.path.join(MIGRATIONS_DIRECTORY, "sqlite_versions")
+SQLITE_REVISION_LIST = os.listdir(SQLITE_REVISIONS_FOLDER)
+SQLITE_REVISION_IDS = []
+if len(SQLITE_REVISION_LIST) > 0:
```

Comment:
No need for this in Python - if there is nothing in the list then the for loop below just won't run at all
(Same applies fo the Postgres equivalent on L60)